src/main/java/com/github/onsdigital/perkin/helper/SdxStore.java

```diff
@@ -0,0 +1,34 @@
+package com.github.onsdigital.perkin.helper;
+
+import com.github.davidcarboni.httpino.Endpoint;
+import com.github.davidcarboni.httpino.Host;
+import com.github.davidcarboni.httpino.Http;
+import com.github.davidcarboni.httpino.Response;
+import com.github.onsdigital.ConfigurationManager;
+import com.github.onsdigital.perkin.json.Result;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+
+/**
+ * Store JSON Survey via HTTP.
+ */
+@Slf4j
+public class SdxStore {
+
+    private String host;
+    private String path;
+
+    public SdxStore() {
+        host = ConfigurationManager.get("STORE_HOST");
+        path = ConfigurationManager.get("STORE_PATH");
+    }
+
+    public Response<Result> store(final String json) throws IOException {
+
+        Endpoint endpoint = new Endpoint(new Host(host), path);
+        log.info("STORE|storing survey to endpoint: {}", endpoint);
+
+        return new Http().postJson(endpoint, json, Result.class);
```

Comment:
The postJson method within Http() in Httpino has a bug in it which returns a quoted string, meaning that the strings are all rejected by validate. I've investigated and postString doesn't have this problem (as we're using it on decrypt). You can see we're we've had to make a similar change in the past here: https://github.com/ONSdigital/sdx-collect/commit/51ad12fbd237cff9c976bf04ded40bcde10ded20

Additionally in order to mock out tests, there's a HttpManager class I was using on the decrypt method. If you can use this that means we can can easily create tests for these endpoints, without needing a bunch of dependencies set up.